### ISSUE TITLE
Unable to load Social app after update to 0.2.2

### ISSUE THREAD

**Issue by N07070** [nextcloud/social#571]
**Describe the bug**
The page is blank, with the exception of the nav bar.

**To Reproduce**
1. Update to 0.2.2
2. Click on the social app icon.
3. See error

**Expected behavior**
Use the social app.

**Client details:**
 - OS: Debian 9
 - Browser: Firefox
 - Version: 60.6.3esr 
 - Device: Desktop

<details>

**Social app version:** 0.2.2

**Web server:** Ngninx

**Database:** MYSQL

**PHP version:** 9

**Nextcloud version:** 15

#### Nextcloud log (data/nextcloud.log)
```
{"reqId":"jXK4G3izEkjlGzkrbpD4","level":2,"time":"2019-06-06T15:42:30+00:00","remoteAddr":"xxx","user":"--","app":"no app in context","method":"POST","url":"/apps/social/inbox","message":"500 - {\"status\":-1,\"exception\":\"OCA\\\\Social\\\\Exceptions\\\\ItemUnknownException\",\"message\":\"\"}","userAgent":"http.rb/3.3.0 (Mastodon/2.8.4; +https://dedezzf.party/)","version":"15.0.8.1","id":"5cf9445cd8ce8"}
```

#### Browser log
```
Content Security Policy: Directive ‘child-src’ has been deprecated. Please use directive ‘worker-src’ to control workers, or directive ‘frame-src’ to control frames respectively.
JQMIGRATE: Migrate is installed, version 1.4.0
core.js:7:542
ReferenceError: regeneratorRuntime is not defined[Learn More]
social.js:127:19804
Source map error: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Resource URL: https://cloud.xyz/apps/social/js/social.js?v=0ea87a65-8
Source Map URL: social.js.map[Learn More]
```

Thanks for your work  !

**Comment by laarisyko** [comment]
I got the same issue here. I have Nextcloud 15 and I wanted to try Nextcloud social. I have installed over the app manager. If I click on the icon, I just get a blank page. Would love to try this app! This is a great idea to integrate nextcloud in this network.

**Comment by daita** [comment]
Do you have any error code other than 200 in the 'Network' tab of the Developer Tools of your browser ?

**Comment by daita** [comment]
Please confirm issue is fixed in 0.2.5

**Comment by N07070** [comment]
Fixed and working ! Thanks a lot for the quick fix !
